pyaedt/edb_core/components.py

```diff
@@ -400,7 +402,189 @@ def get_solder_ball_height(self, cmp):
         return False
 
     @aedt_exception_handler
-    def set_solder_ball(self, cmp, sball_height=100e-6, sball_diam=150e-6, orientation=FlipChipOrientation.Up):
+    def create_port_on_component(self, cmp, net_list, port_type=SourceType.CoaxPort, do_pingroup=True, refnet="gnd"):
```

Comment:
If/when possible, I would encourage to avoid any short cuts for parameters' name.
```suggestion
    def create_port_on_component(self, component, net_list, port_type=SourceType.CoaxPort, do_pingroup=True, reference_net="gnd"):
```